PacketDotNet/IPv6Packet.cs

```diff
@@ -171,23 +183,34 @@ public override Int32 TotalLength
         }
 
         /// <summary>
-        /// Identifies the protocol encapsulated by this packet
-        /// Replaces IPv4's 'protocol' field, has compatible values
+        /// Identifies the NextHeader, will be the protocal of the encapsulated in this ip packet unless there are extended headers
         /// </summary>
-        public override IPProtocolType NextHeader
```

Comment:
undo I think

Reply:
No this is ok, I've moved the NextHeader into IPv6Packet from IPPacket as IPv4 doesn't have the same concept